app/App.hs

```diff
@@ -37,3 +46,25 @@ main = do
             Async.withAsync (HTTP.run events httpServerPort) $ \serverAsync -> do
                 Async.link serverAsync
                 Streaming.mainLoop events config points
+
+generateScriptsExample :: IO ()
+generateScriptsExample = do
+    let path = "scripts.yaml"
+    fileExists <- doesFileExist path
```

Comment:
We should just print it to `stdout`. More control and predictable behaviour for the user.